packages/rancher-istio/charts/Chart.yaml

```diff
@@ -13,6 +13,6 @@ annotations:
   catalog.cattle.io/release-name: rancher-istio
   catalog.cattle.io/ui-component: istio
   catalog.cattle.io/provides-gvr: networking.istio.io.virtualservice/v1beta1
-  catalog.cattle.io/auto-install: rancher-kiali-server-crd=1.24.003
+  catalog.cattle.io/auto-install: rancher-kiali-server-crd=1.29.000
```

Comment:
Doesn't this have to change in requirements.txt too ? Does your script still work ?

Reply:
This is part of the changes that we had to make with the new scripts. The version for dependencies now comes from `generated-changes/dependencies/<kiali>/dependency.yaml`

Reply:
Ok, can we remove requirements.txt ?

Reply:
The requirements.txt turned into requirements.yaml and it needs to remain for the pulling in the correct generated dependencies https://github.com/aiyengar2/charts-build-scripts/blob/a26221f4b798e9df62439b96d2c0c9c2acc3dbe7/templates/template/scripts/regenerate-packages#L100

Reply:
Ok, so just the script/version_update needs updated. We can do that a diff time but try to hit eventually.

Reply:
@brendarearden if you do `make charts` and then pull it into rancher and install, is it able to find the CRD chart to auto-install?

Just realized that if the CRD chart has `-rc00` appended to it the scripts may not be able to find it on the UI to auto-install. Wondering what happens in this case.

At worst case, this would only impact QA being able to install the CRD chart stuff. Should still be fine when we actually release, but the rc version in this annotation will need to be modified in `make charts`.

Reply:
It is not able to find the CRD chart for auto install so this will need to be adjusted for testing RCs

Reply:
Since we need to get these to QA asap, we should just take out the RC stuff and do version bumps directly. If it fails QA and we need to update again we can do another version bump or overwrite the current version and make QA aware of that. cc @brendarearden / @aiyengar2

Reply:
@brendarearden once https://github.com/rancher/charts/pull/964, https://github.com/rancher/charts/pull/965, https://github.com/rancher/charts/pull/966 are in (which vendor in changes to scripts for https://github.com/rancher/charts-build-scripts/pull/10), you will need to slightly modify the PR to set this annotation to `rancher-kiali-server-crd=1.29.000-rc00`.

Since the releaseCandidateVersion of the CRD chart of rancher-kiali-server is not tied to the releaseCandidateVersion of the rancher-istio chart, the fix in the scripts is to just to ensure that the rcVersion gets dropped when we sync to main (tested this out locally and it seems to work fine).

You'll still need to make sure you update the Chart annotation manually any time a new release candidate version comes in for rancher-kiali-server.